saleor/graphql/views.py

```diff
@@ -202,8 +202,14 @@ def format_error(error):
 
         if settings.DEBUG:
             lines = []
-            for line in traceback.format_exception(type(exc), exc, exc.__traceback__):
-                lines.extend(line.rstrip().splitlines())
+
+            # FIXME: test :80
```

Comment:
What do you mean? :)

Reply:
It means: "@NyanKiyoshi please add test runs the line 80". It was raising an internal error (`DEBUG = True` only) when input was invalid because there was no trace. I forgot to add the test... That's why there was a fixme because... I knew I would forget :wink: